WordPress/Classes/ViewRelated/Domains/Domain registration/RegisterDomainSuggestions/RegisterDomainCoordinator.swift

```diff
@@ -0,0 +1,195 @@
+import Foundation
+
+class RegisterDomainCoordinator {
+
+    // MARK: Type Aliases
+
+    typealias DomainPurchasedCallback = ((UIViewController, String) -> Void)
+    typealias DomainAddedToCartCallback = ((UIViewController, String) -> Void)
+
+    // MARK: Variables
+
+    var site: Blog?
+    var domainPurchasedCallback: DomainPurchasedCallback?
+    var domainAddedToCartCallback: DomainAddedToCartCallback?
+    var domain: FullyQuotedDomainSuggestion?
```

Comment:
❓  Are `site` and `domain` properties mutated at some point in the lifecycle of `RegisterDomainCoordinator` object?

Reply:
Yes
* `domain` is set in `RegisterDomainSuggestionsViewController.domainSelected()`
* `site` will be set in a future PR when the site picker is implemented